code/datums/mood_events/generic_positive_events.dm

```diff
@@ -184,3 +184,8 @@
 /datum/mood_event/area/add_effects(_mood_change, _description)
 	mood_change = _mood_change
 	description = _description
+
+/datum/mood_event/hydration
+	description = "<span class='nicegreen'>I feel healthy and refreshed!</span>\n"
+	mood_change = 4
+	timeout = 6 MINUTES
```

Comment:
this is stronger than mixed drinks, come on

Reply:
Not quite but I do agree with you after looking at the stats. Also, drinks that have alcohol have an additional +2 for drunkness. Ill not it down a point so its on par with nice tasting drinks.

Reply:
it should be something on par with hug in terms of duration, it's literally just drinking water which is infinite in sinks.